src/components/Button/Button.scss

```diff
@@ -487,7 +487,7 @@ $stacking-order: (
 
   // Because the outline border color has a 40% opacity, the left border appears darker than the rest of the borders because they are layered over one another. Reducing the opacity to zero for the connected disclosure when not focused gives us the expected border color.
   &.outline:not(:focus) {
-    border-left-color: rgba(color('ink', 'lighter'), 0);
+    border-left-color: rgba(var(--p-border-disabled), 0);
```

Comment:
I believe rgba is a sass function, I don't think it will have the desired effect when you pass in a css variable, as the color won't be resolved till runtime

Reply:
I didn't know this! Thanks @BPScott 

@alex-page what's the best way forward on this? hard code the color? leave for now and create a new issue? is `rgba()` going to be removed eventually as well?

Reply:
Lets hard code the color to unblock this PR. It will be a failure on `polaris-coverage` so we won't miss it.